test/index.js

```diff
@@ -1,8 +1,10 @@
 'use strict'
 
-const Lab = require('lab')
+require('dotenv').config()
```

Comment:
I bring in `dotenv` to read the DATABASE_URL if not passed in. It is a dev-only dependency.